<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarangKeluarForm</class>
 <widget class="QWidget" name="BarangKeluarForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Barang Keluar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size:20px; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Transaksi Barang Keluar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelUserAktif">
     <property name="text">
      <string>User Aktif: -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBarang">
       <property name="text">
        <string>Barang</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBarang"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTanggal">
       <property name="text">
        <string>Tanggal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="input_tanggal"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelJumlah">
       <property name="text">
        <string>Jumlah</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="input_jumlah"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUbah">
       <property name="text">
        <string>Ubah</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHapus">
       <property name="text">
        <string>Hapus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBersih">
       <property name="text">
        <string>Bersih</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_cari">
     <property name="placeholderText">
      <string>Cari berdasarkan kode atau nama..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableBarangKeluar">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>ID Keluar</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama Barang</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tanggal</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Jumlah</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
